README.md

```diff
@@ -1 +1,143 @@
-# budget-app
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 💻 Budget-APP <a name="about-project"></a>
+Budget app is mobile first application which is allow users to track their purchases. Using this budget app users can manage their purchases easily. User can add and remove new category and also can add purchases to it. They can see their total parchases with total amount.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<summary>FrontEnd</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JavaScript</li>
+  </ul>
+
+<summary>BackEnd</summary>
+<ul>
+  <li>Ruby</li>
+</ul>
+
+<summary>Frameworks</summary>
+<ul>
+  <li>Rails</li>
+</ul>
+
+<summary>Database</summary>
+<ul>
+  <li>PostgreSQL</li>
+</ul>
+
+### Key Features <a name="key-features"></a>
+
+- **User can manage their category.**
+- **User can purchase new item with multiple categories.**
+- **User can remove purchase**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Project Demo <a name="live-demo"></a>
+
+### Project Preview
+![c1](https://github.com/raihan2bd/budget-app/assets/35267447/00a0ce20-0406-471b-a734-0409f9bcdc67)
+![c2](https://github.com/raihan2bd/budget-app/assets/35267447/da43a9f1-f5c3-4c0c-aef1-9dc5f8eabc03)
+![c3](https://github.com/raihan2bd/budget-app/assets/35267447/4075bb92-d9c4-4d30-81a9-9008784675d0)
+
+
+### [Live Project Demo](https://budget-app-ktyf.onrender.com)
```

Comment:
- [ ] When I visit your live demo, input details to sign up, and click the sign-up button the app gives me an error instead. Kindly look into your server logs and fix the issue.

  ![image](https://github.com/raihan2bd/budget-app/assets/36542986/fc913fee-3a25-454a-b1a8-83e140939364)

Reply:
Thanks once again for your information. However, sometimes it takes time to load the server especially on `render` Would you like to check it again because it is working on my side 🙂 
![Capturesss](https://github.com/raihan2bd/budget-app/assets/35267447/5f6676d4-d87c-4c52-b9e3-e0e113d4fdfe)

Reply:
Hi @raihan2bd this is not the screen I am referring to. I mean after filling out the sign up form and clicking "create an account", I get the error. I checked again, same issue.

![image](https://github.com/raihan2bd/budget-app/assets/36542986/fc7b8812-8678-46a5-baae-3ffec2d6d430)

Reply:
Thanks @TedLivist for your valuable information I will fix it before the next review 👍